docs/news/upgrades.rst

```diff
@@ -130,6 +130,17 @@ Inventory variable changes
   been redesigned to use YAML lists instead of dictionaries. See
   :ref:`netbase__ref_hosts` for more details.
 
+- The ``resources__group_name`` variable has been removed in favor of using
+  all the groups the current hosts is in. This change has been reflected in the
+  updated variable ``resources__group_templates``.
```

Comment:
You can create internal references directly to the variables like this:
```rst
:envvar:`resources__group_templates`
```
Of course this doesn't work for variables that are removed.